services/app-web/src/components/FileUpload/FileProcessor/FileProcessor.tsx

```diff
@@ -223,7 +223,6 @@ export const FileProcessor = ({
             hasRecoverableError={hasRecoverableError}
             handleDelete={handleDelete}
             handleRetry={handleRetry}
```

Comment:
Are we removing the table mode fully or not? If we are this is fine but then I would expect everything associated with the table `renderMode` on the top level fileupload to go. Otherwise I would think this checkbox stuff stays and just the feature flag is removed from the code consuming this component.

Reply:
I forgot to, I'll add it in my pr response commit